smoke/features/jenkins-nodejs-agent.feature

```diff
@@ -11,5 +11,6 @@ Feature: Testing jenkins agent nodejs image
     Given The jenkins pod is up and runnning
     When The user enters new-app command with nodejs_template
     Then Trigger the build using oc start-build
-    Then nodejs-postgresql-example pod must come up
+    Then verify the build status of "nodejs-postgresql-example-1" build is Complete
+    Then verify the build status of "nodejs-postgresql-example-2" build is Complete
```

Comment:
unclear how this is related to the persistence template

did you just decide to add more verification, regardless of persistent vs. ephemeral?

if so, I think the PR title should capture that.

If this is somehow related to persistence, please clarify.

Reply:
This test the agent images on jenkins installed using persistent template. O already have test of agent on ephemeral templates. But wanted to make sure the agents are tested on persistent templates too

Reply:
ok thanks for clarifying ... I'm good with things as is